### DISCUSSION TITLE
Cutting power to a NooElec NESDR SMArTee v2 SDR

### DISCUSSION THREAD

**Discussion by clarkerz** [mvp/uhubctl#335] [Q&A] [answered]
I installed on my RPi4B (model d03114) and checked the EPROM state (is 000138a1) but even after executing sudo uhubctl -a off -p 4 -l 1-1, and the output states  the power is off to the USB port powering my NooElec NESDR SMArTee v2 SDR, it is still powering the LNA conencted to it via coax. Just checking is expected behavirout? Cheers.

PS. Thanks for the software BTW.

Andy

**Comment by mvp** [comment, marked as answer]
This is FAQ. Raspberry Pi 4B requires to turn off all 4 ports to actually cut power off:

    uhubctl -l 1-1 -a 0

Not specifying port number is equivalent to `-p all`

**Reply by clarkerz** [reply]
Ah many thanks I missed that...nice one.
